communications/mongo_models.py

```diff
@@ -0,0 +1,13 @@
+from mongoengine import Document, ListField, StringField, IntField, DateTimeField, ReferenceField
+from datetime import datetime
+
+class Room(Document):
+    participants = ListField(IntField(), required=True)  
+    created_at = DateTimeField(default=datetime.now)
+    updated_at = DateTimeField(default=datetime.now)
```

Comment:
Use `datetime.utcnow` instead of `datetime.now` to avoid timezone issues and ensure consistency